### DISCUSSION TITLE
Visualize ebusd data

### DISCUSSION THREAD

**Discussion by Commifreak** [john30/ebusd#667] [Q&A]
Hi,

I want to visualize the ebusd data somehow. Is using telegraf -> Influx -> Grafan the easiest/best way? Any other ideas?
I would also go the way: /data (json) -> mysql -> Grafana, any experiences here?

Thanks! :)

**Comment by m0deller** [comment]
I use emoncms. Has lot of visualisation capabilities. 
ebus adapter->ebusd->MQTT->NodeRed->MQTT->emoncms

**Comment by cyberthom42** [comment]
For Visualization I am using Grafana, here an example:
![image](https://user-images.githubusercontent.com/115342533/206519911-fd5d5c3c-53fd-45f6-a050-d9d56ae1be04.png)
To poll the ebus data I am using iobroker, the objects are written to an influxdb.

**Reply by lkoniecki** [reply]
Can you share your Grafana's dashboard source?

**Reply by svn2208** [reply]
I am also intrested.
